custom_navbutton_completion.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
I'd generally prefer not to be introducing new code that does not follow up to date guidelines, but if you're offering to fix this with a later patch, then I'll let it in for now.